djes/mapping.py

```diff
@@ -5,20 +5,37 @@
 
 from djes.conf import settings
 
-# TODO: expand this for all django field types
 FIELD_MAPPINGS = {
     "AutoField": {"type": "long"},
-    "OneToOneField": {"type": "long"},
-    "ForeignKey": {"type": "long"},
-    "ManyToManyField": {"type": "long"},
-    "IntegerField": {"type": "long"},
+    "BigIntegerField": {"type": "long"},
+    "BinaryField": {"type": "binary"},
+    "BooleanField": {"type": "boolean"},
     "CharField": {"type": "string"},
-    "TextField": {"type": "string"},
-    "SlugField": {"type": "string", "index": "not_analyzed"},
-    "DateTimeField": {"type": "date"},
+    "CommaSeparatedIntegerField": {"type": "string"},
```

Comment:
I converted this to an integer locally and rewrote the tests to match. The issue is the introspection for the field comes up as `CharField` in the mapping.

I'm going to leave this as is for now and open a ticket to work on this later.